src/pudl/etl.py

```diff
@@ -187,41 +187,48 @@ def _etl_eia(etl_params, datapkg_dir, pudl_settings):
 
     """
     eia_inputs = _validate_params_eia(etl_params)
-    eia923_tables = eia_inputs['eia923_tables']
-    eia923_years = eia_inputs['eia923_years']
-    eia860_tables = eia_inputs['eia860_tables']
-    eia860_years = eia_inputs['eia860_years']
-
-    if (not eia923_tables or not eia923_years) and \
-            (not eia860_tables or not eia860_years):
+    eia860_tables = eia_inputs["eia860_tables"]
+    eia860_years = eia_inputs["eia860_years"]
+    eia861_tables = eia_inputs["eia861_tables"]
+    eia861_years = eia_inputs["eia861_years"]
+    eia923_tables = eia_inputs["eia923_tables"]
+    eia923_years = eia_inputs["eia923_years"]
+
+    if (
+        (not eia923_tables or not eia923_years)
+        and (not eia860_tables or not eia860_years)
+        and (not eia861_tables or not eia861_years)
+    ):
         logger.info('Not loading EIA.')
         return []
 
     # generate CSVs for the static EIA tables, return the list of tables
     static_tables = _load_static_tables_eia(datapkg_dir)
 
     # Extract EIA forms 923, 860
-    data_dir = pudl_settings["data_dir"]
-    eia923_raw_dfs = pudl.extract.eia923.Extractor(
-        data_dir).extract(eia923_years)
-    eia860_raw_dfs = pudl.extract.eia860.Extractor(
-        data_dir).extract(eia860_years)
+    eia860_raw_dfs = pudl.extract.eia860.Extractor().extract(eia860_years, testing=True)
+    eia861_raw_dfs = pudl.extract.eia861.Extractor().extract(eia861_years, testing=True)
+    eia923_raw_dfs = pudl.extract.eia923.Extractor().extract(eia923_years, testing=True)
     # Transform EIA forms 923, 860
-    eia923_transformed_dfs = pudl.transform.eia923.transform(
-        eia923_raw_dfs, eia923_tables=eia923_tables)
     eia860_transformed_dfs = pudl.transform.eia860.transform(
         eia860_raw_dfs, eia860_tables=eia860_tables)
+    eia861_transformed_dfs = pudl.transform.eia861.transform(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected keyword argument 'eia860_tables' in function call](https://app.codacy.com/manual/zaneselvans/pudl/pullRequest?prid=5664147)